libraries/ini_helper.rb

```diff
@@ -1,5 +1,15 @@
 module GrafanaCookbook
   module IniHelper
+    def self.format_multi_config(multi_config)
+      output = []
+      i=0
```

Comment:
It's a hash and you're going to iterate from 0 to N .. that seems error prone.
If we keep it a hash, maybe iterate over everything ?